addon/components/bs-form/element.d.ts

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+
+interface BsFormElementSignature<T> {
+  Element: HTMLDivElement;
+  Args: {
+    controlType?: 'text' | 'checkbox' | 'radio' | 'switch' | 'textarea' | string;
+    customError?: 'string';
+    customWarning?: string;
+    doNotShowValidationForEventTargets?: unknown;
```

Comment:
Per [API docs](https://www.ember-bootstrap.com/api/classes/Components.FormElement.html#property_doNotShowValidationForEventTargets):

```suggestion
    doNotShowValidationForEventTargets?: string[] | null;
```